### ISSUE TITLE
Better control panel for sos-report (and sos-report-toc) template.

### ISSUE THREAD

**Issue by BoPeng** [vatlab/sos-notebook#123]
We have a hidden [control panel]((https://github.com/vatlab/sos-notebook/blob/master/src/sos_notebook/templates/parts/control_panel.tpl#L51)) to control the display of cells and outputs in `sos-report` and `sos-report-toc` templates. The panel is completely hidden and can lead to seemingly empty page when there is no markdown cell in the notebook. It would be helpful to change it to a dropdown button or top bar so that it can be more visible. Also, when a toc is displayed, it could be integrated to the toc (e.g. on the header).

**Comment by j-andrews7** [comment]
Yes, this would be nice, particularly if trying to share the report with other people who seem incapable of finding the panel even with explicit instructions.

**Comment by BoPeng** [comment]
Finally, with #171 out of the way, I can work on a better control panel. Actually, the example https://willwerscheid.github.io/MASHvFLASH/MASHvFLASHnn2.html that @gaow proposed looks pretty simple and perhaps good enough.

**Comment by BoPeng** [comment]
Well, the details can be improved but the bulk of the code is in. Basically the original control panel is changed to a drop down box.
